src/linear_algebra/alternating.lean

```diff
@@ -249,6 +249,9 @@ instance : has_scalar S (alternating_map R M N ι) :=
 @[norm_cast] lemma coe_smul (c : S):
   ((c • f : alternating_map R M N ι) : multilinear_map R (λ i : ι, M) N) = c • f := rfl
 
+lemma coe_fn_smul (c : S) (f : alternating_map R M N ι) : ⇑(c • f) = c • ⇑f :=
+funext $ λ _, rfl
```

Comment:
Is this one not also `rfl`? I ask, because if a lemma is proved by `rfl`, then it's allowed to be used in `dsimp`.